### PR TITLE
Add values for mapper cloud connection

### PR DESCRIPTION
**Pull request by NetanelBollag**
### Description

This PR add helm support for cloud connection configuration.

### CODE REVIEW

File: network-mapper/templates/mapper-deployment.yaml [hunk 1]
```diff
@@ -25,6 +25,16 @@ spec:
           env:
             - name: OTTERIZE_DEBUG
               value: {{ .Values.debug | quote }}
+            - name: OTTERIZE_CLOUD_API_ADDRESS
```
Comment (orishoshan) [inline network-mapper/templates/mapper-deployment.yaml:28]: Please add docs and use the correct case (javaScriptStyle)

File: network-mapper/templates/mapper-deployment.yaml [hunk 2]
```diff
@@ -25,6 +25,16 @@ spec:
...
+              value: {{ .Values.mapper.cloudApiAddress | "https://app.otterize.com/api" }}
+            - name: OTTERIZE_CLIENT_ID
+              value: {{ .Values.mapper.clientId | quote }}
+            - name: OTTERIZE_CLIENT_SECRET
+              value: {{ .Values.mapper.clientSecret | quote }}
+            - name: OTTERIZE_UPLOAD_INTERVAL_SECONDS
+              value: {{ .Values.mapper.uploadIntervalSeconds | "60" }}
+            - name: OTTERIZE_CLOUD_GRAPHQL_ENDPOINT
```
Comment (orishoshan) [inline network-mapper/templates/mapper-deployment.yaml:36]: I didn't notice that this was separate until I saw it in the docs - please put it into the API_ADDRESS variable so it's the same as the CLI and the intents operator.

File: network-mapper/templates/mapper-deployment.yaml (hunk 1 above)
Comment (orishoshan) [inline network-mapper/templates/mapper-deployment.yaml:28]: This variable should be named OTTERIZE_API_ADDRESS, so it's the same name as the CLI configuration. Need to change it in the network mapper itself too. It should look like this: https://github.com/otterize/intents-operator/blob/develop/src/shared/otterizecloud/otterizecloudclient/cloud_client.go#L16

Comment (NetanelBollag) [reply network-mapper/templates/mapper-deployment.yaml]: Did it https://github.com/otterize/network-mapper/pull/52

File: network-mapper/README.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
-| `mapper.pullPolicy`       | Mapper pull policy.      | `(none)`         |
-| `mapper.resources`        | Resources override.      | `(none)`         |   
+| Key                            | Description                          | Default                        |
+|--------------------------------|--------------------------------------|--------------------------------|
+| `mapper.image.repository`      | Mapper image repository.             | `otterize`                     |
+| `mapper.image.image`           | Mapper image.                        | `network-mapper`               |
+| `mapper.image.tag`             | Mapper image tag.                    | `latest`                       |
+| `mapper.pullPolicy`            | Mapper pull policy.                  | `(none)`                       |
+| `mapper.resources`             | Resources override.                  | `(none)`                       |
+| `mapper.apiAddress`            | Api address.                         | `https://app.otterize.com/api` |
```
Comment (orishoshan) [inline network-mapper/README.md:11]: API should be uppercase

File: network-mapper/README.md [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+| `mapper.clientSecret`          | Cloud client secret.                 | `(none)`                       |
+| `mapper.clientid`              | Cloud client id.                     | `(none)`                       |
```
Comment (orishoshan) [inline network-mapper/README.md:13]: These settings should be for the entire chart, not the mapper - it's an implementation detail that it's just the mapper.
Also, the docs should say something like - Client ID for connecting to Otterize Cloud, Client Secret for connecting to Otterize.

**Pull request merged**